pkg/util/ready/ready.go

```diff
@@ -0,0 +1,56 @@
+package ready
```

Comment:
I wonder if these should be moved here eventually: https://github.com/Azure/ARO-RP/blob/4830472f6f7b49df44b6bff623e53a81f884b135/pkg/install/condition.go